vcd/resource_vcd_org_vdc.go

```diff
@@ -972,8 +972,8 @@ func changeComputePoliciesAndDefaultId(d *schema.ResourceData, vcdClient *VCDCli
 	for _, attribute := range computePolicyAttributes {
 		vmComputePolicyIds = append(vmComputePolicyIds, convertSchemaSetToSliceOfStrings(d.Get(attribute).(*schema.Set))...)
 	}
-	if !contains(vmComputePolicyIds, defaultPolicyId.(string)) {
-		return fmt.Errorf("`default_compute_policy_id` %s is not present in any of `%v`", defaultPolicyId.(string), computePolicyAttributes)
+	if defaultPolicyId.(string) != "" && !contains(vmComputePolicyIds, defaultPolicyId.(string)) {
```

Comment:
We could add a comment above this line, explaining the rationale and even mentioning the issue number.

Reply:
Indeed, that would help. Added